### PR TITLE
add filtering instructions to ILogger getting started doc

### PR DESCRIPTION
**Pull request by TimothyMothra**
Fixes #.

## Changes

To configure filtering rules, a user must know either the class name `OpenTelemetryLoggingProvider` or it's alias "OpenTelemetry".
This change adds this to the doc and gives concrete configuration examples.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/logs/getting-started/README.md [hunk 1]
```diff
@@ -53,13 +53,48 @@ What does the above program do?
...
 [`ILogger`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
-API to log a formatted string with a severity level of Information. Click
-[here](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.loglevel)
-for more information on the different logs levels. Opentelemetry captures this
-and sends it to `ConsoleExporter`. `ConsoleExporter` simply displays it on the
-console.
+API to log a formatted string with a severity level of Information. OpenTelemetry
+captures this and sends it to the `ConsoleExporter` which displays logs on the console.
+
+## Configure Filtering
```
Comment (reyang) [inline docs/logs/getting-started/README.md:59]: Why is this needed in a "getting started doc"?

Comment (cijothomas) [reply docs/logs/getting-started/README.md:59]: Lets stick with the doc title - "Getting started in 5 minutes", so anything not required to get up and running, can be added elsewhere.
